server/chat-formatter.ts

```diff
@@ -458,11 +439,9 @@ class TextFormatter {
 
 /**
  * Takes a string and converts it to HTML by replacing standard chat formatting with the appropriate HTML tags.
- *
- * @param {string} str
  */
-function formatText(str, isTrusted = false) {
+function formatText(str: string, isTrusted: boolean = false) {
 	return new TextFormatter(str, isTrusted).get();
 }
 
-module.exports = {formatText, linkRegex};
+export = {formatText, linkRegex};
```

Comment:
You should probably export `formatText` and `linkRegex` directly in their declaration.